<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>-40</y>
      <width>941</width>
      <height>531</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../paw 1.jpg</pixmap>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>541</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;Adobe Caslon Pro Bold&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Ajouter un animal</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Adobe Caslon Pro Bold&quot;;</string>
      </property>
      <property name="text">
       <string>identifiant</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Adobe Caslon Pro Bold&quot;;</string>
      </property>
      <property name="text">
       <string>esp√®ce</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Adobe Caslon Pro Bold&quot;;</string>
      </property>
      <property name="text">
       <string>sexe</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Adobe Caslon Pro Bold&quot;;</string>
      </property>
      <property name="text">
       <string>race</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Adobe Caslon Pro Bold&quot;;</string>
      </property>
      <property name="text">
       <string>poids</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Adobe Caslon Pro Bold&quot;;</string>
      </property>
      <property name="text">
       <string>date d'accueil</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style:outset;
border-width:2px;
border-radius:10px;
</string>
      </property>
      <property name="maxLength">
       <number>8</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_espece">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style:outset;
border-width:2px;
border-radius:10px;
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_sexe">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style:outset;
border-width:2px;
border-radius:10px;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_race">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style:outset;
border-width:2px;
border-radius:10px;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_poids">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style:outset;
border-width:2px;
border-radius:10px;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>200</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:black;
color:white;
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:white;
</string>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>200</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:black;
color:white;
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:white;</string>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_date">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>200</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style:outset;
border-width:2px;
border-radius:10px;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_age">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style:outset;
border-width:2px;
border-radius:10px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Adobe Caslon Pro Bold&quot;;</string>
      </property>
      <property name="text">
       <string>age</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Afficher un animal</string>
     </attribute>
     <widget class="QTableView" name="table_animaux">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>241</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_espece_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>40</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style:outset;
border-width:2px;
border-radius:10px;
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_age_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>130</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style:outset;
border-width:2px;
border-radius:10px;
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_race_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>100</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style:outset;
border-width:2px;
border-radius:10px;
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_sexe_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>70</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style:outset;
border-width:2px;
border-radius:10px;
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_poids_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>160</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style:outset;
border-width:2px;
border-radius:10px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_date_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>190</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style:outset;
border-width:2px;
border-radius:10px;
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Adobe Caslon Pro Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Esp√©ce</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>70</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Adobe Caslon Pro Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Sexe</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>100</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Adobe Caslon Pro Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Race</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>130</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Adobe Caslon Pro Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>160</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Adobe Caslon Pro Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Poids</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>190</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 10pt &quot;Adobe Caslon Pro Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Date d'accueil</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_modifier">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:black;
color:white;
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:white;</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style:outset;
border-width:2px;
border-radius:10px;</string>
      </property>
      <property name="maxLength">
       <number>8</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ok">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>230</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:black;
color:white;
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:white;</string>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id_3">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-style:outset;
border-width:2px;
border-radius:10px;
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>ID :</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Supprimer un animal</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>381</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>suppression</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;Adobe Caslon Pro Bold&quot;;</string>
       </property>
       <property name="text">
        <string>identifiant :</string>
       </property>
      </widget>
      <widget class="QPushButton" name="supp">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>120</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:black;
color:white;
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:white;</string>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
      <widget class="QComboBox" name="del_id">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>80</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-style:outset;
border-width:2px;
border-radius:10px;
</string>
       </property>
      </widget>
      <widget class="QPushButton" name="load_list">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:black;
color:white;
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:white;</string>
       </property>
       <property name="text">
        <string>Load list</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Modifier un animal</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>908</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
